bitnami/etcd/values.yaml

```diff
@@ -23,271 +23,233 @@ image:
-## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+## Add labels to all the deployed resources
 ##
-hostAliases: []
+commonLabels: {}
 
-## Init containers parameters:
-## volumePermissions: Change the owner and group of the persistent volume mountpoint to runAsUser:fsGroup values from the securityContext section.
+## Add annotations to all the deployed resources
 ##
-volumePermissions:
-  enabled: false
-  image:
-    registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: "10"
-    pullPolicy: Always
-    ## Optionally specify an array of imagePullSecrets.
-    ## Secrets must be manually created in the namespace.
-    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-    ##
-    # pullSecrets:
-    #   - myRegistryKeySecretName
-  ## Init container' resource requests and limits
-  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
-  ##
-  resources:
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    limits: {}
-    #   cpu: 100m
-    #   memory: 128Mi
-    requests: {}
-    #   cpu: 100m
-    #   memory: 128Mi
+commonAnnotations: {}
 
-## Statefulset parameters
+## Force target Kubernetes version (using Helm capabilities if not set)
 ##
-statefulset:
-  ## Number of replicas
-  ##
-  replicaCount: 1
-  ## Update strategy, can be set to RollingUpdate or OnDelete by default.
-  ## https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets
-  ##
-  updateStrategy: RollingUpdate
-  ## Partition update strategy
-  ## https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions
-  ##
-  # rollingUpdatePartition:
-  ## Pod management policy
-  ## https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
-  ##
-  podManagementPolicy: Parallel
-
-## ConfigMap that includes the etcd.conf.yml file
-##
-# configFileConfigMap:
+kubeVersion: ""
 
-## ConfigMap that includes extra environment variables
-##
-# envVarsConfigMap:
-
-## Allow to use etcd without configuring RBAC authentication
+## Kubernetes Cluster Domain
 ##
-allowNoneAuthentication: true
+clusterDomain: cluster.local
 
-## Limits the number of operating system threads that can execute user-level
-## Go code simultaneously by setting GOMAXPROCS environment variable
-## ref: https://golang.org/pkg/runtime
+## Extra objects to deploy (value evaluated as a template)
 ##
-# maxProcs:
+extraDeploy: []
 
-## Authentication parameteres
-## https://github.com/bitnami/bitnami-docker-etcd#security
+## Authentication parameters
 ##
 auth:
```

Comment:
No mention to `auth.client.clientPortNameOverride` nor `auth.client.peerPortNameOverride`

Reply:
If it's a deprecated value, then mention it in the notable changes

Reply:
It was a typo